tools/eslint-rules/prefer-assert-methods.js

```diff
@@ -24,15 +13,12 @@ const preferedAssertMethod = {
 
 module.exports = function(context) {
   return {
-    ExpressionStatement(node) {
-      if (isAssert(node)) {
-        const arg = getFirstArg(node.expression);
-        if (arg && arg.type === 'BinaryExpression') {
-          const assertMethod = preferedAssertMethod[arg.operator];
-          if (assertMethod) {
-            context.report(node, parseError(assertMethod, arg.operator));
-          }
-        }
+    // eslint-disable-next-line max-len
+    'ExpressionStatement[expression.type="CallExpression"][expression.callee.name="assert"][expression.arguments.0.type="BinaryExpression"]': function(node) {
```

Comment:
For the sake of legibility, we could extract this outside of `module.exports` and just use `[astSelector]`? Your call but I'm not a fan of the run-on line (having the [] beneath each other is a bit more legible IMHO).